olm_deploy/operatorregistry/catalog-source.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: operators.coreos.com/v1alpha1
```

Comment:
@vparfonov  This source is only used during testing.  You will need to learn how to resolve it generally in production if the issue is related to a production release

Reply:
ok, thanks